### ISSUE TITLE
Webpack cannot resolve `process` in storybook 6.4.13

### ISSUE THREAD

**Issue by jackw** [storybookjs/storybook#17285]
**Describe the bug**

Whilst attempting to bump Storybook from 6.4.9 to 6.4.13 Webpack build errors are occurring stating that `process/browser.js` cannot be resolved. The repo in question is using the storybook-webpack5 builders, yarn Pnp and a custom webpack config.

**To Reproduce**
I apologise for not creating a specific reproduction but I have an [open PR](https://github.com/grafana/grafana/pull/44091) that bumps the repo to 6.4.13 but had to apply changes in [this commit](https://github.com/grafana/grafana/pull/44091/commits/58fa36ec8b08d60e76ae081efd2efdaf7350b068?file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.ts&show-viewed-files=true) for storybook to run locally without issue.

**System**
```
Environment Info:

  System:
    OS: macOS 12.0.1
    CPU: (8) x64 Intel(R) Core(TM) i5-1038NG7 CPU @ 2.00GHz
  Binaries:
    Node: 16.13.0 - ~/.nvm/versions/node/v16.13.0/bin/node
    Yarn: 3.1.1 - /usr/local/bin/yarn
    npm: 8.1.0 - ~/.nvm/versions/node/v16.13.0/bin/npm
  Browsers:
    Chrome: 97.0.4692.71
    Firefox: 95.0.2
    Safari: 15.1
```

**Additional context**
I'm guessing it's related to this [PR](https://github.com/storybookjs/storybook/pull/17249) that was included in the 6.4.13 release.

**Comment by shilman** [comment]
That hasn't been patched to 6.4 yet. If you upgrade to 6.5-alpha does that fix things?

```
npx sb upgrade --prerelease 
```

Let me know and I'll patch it back

**Comment by jackw** [comment]
> That hasn't been patched to 6.4 yet. If you upgrade to 6.5-alpha does that fix things?

Ah really? I took a look at https://github.com/storybookjs/storybook/releases/tag/v6.4.13 where that PR is mentioned. 🤔 

Will take 6.5-alpha for a spin shortly and get back to you. Appreciate the quick response @shilman. 👍

**Comment by jackw** [comment]
> If you upgrade to 6.5-alpha does that fix things?

It most certainly does! 🥳  

Removing the SB config changes made in my PR that replace `ProvidePlugin` in `webpackFinal` and `managerWebpack` then running `npx sb upgrade --prerelease` works like a charm (once I fix a couple of yarn PnP related peer dependency issues).
